packages/availity-react-kit/project/app/Request/stores/requestStore.js

```diff
@@ -0,0 +1,102 @@
+import { action, computed, observable } from 'mobx';
+import { getProviders, getOrganizations, getUser } from '../api/requestApi';
+import map from 'lodash.map';
+
+export class Organization {
+  @observable id
+  @observable name
+  @observable customerId
+
+  constructor(json) {
```

Comment:
Why not use destructuring here (and below)?
```
constructor(({ id = '', name = '', customerId = ''} = {})) {
  this.id = id;
  this.name = name;
  this.customerId = customerId;
}
```

Reply:
good point. I'll update to get rid of all the `||` syntax